ChangeLog.md

```diff
@@ -20,6 +20,8 @@ See docs/process.md for more on how version tagging works.
 
 3.1.42 (in development)
 -----------------------
+- Bump the default minimum Node version from 10.19 to 16.0. To target the
```

Comment:
`The default minimum Node version was bumped`

Also can you add the word "default" to the PR title?

Reply:
Done, but isn't "was bumped" passive voice, which is generally frowned upon in English? I guess just "bump" as in the old text isn't even grammatical...

Reply:
"Bump the default minimum Node version" sounds correct grammatically but that sounds like that text I would write for the PR itself.   i.e. its in the present tense, imperative mood.   e.d. "Rename foo", "Delete X"

For the changelog entry I normally write in the past tense think.   e.g. "foo was renamed", "x was deleted".   So I would expect "was bumped" here... but I could be wrong.   Reading the rest of the changelog it seems to be mostly be in the past.  Should we swith to using the same tense/mood for PR and for the changelog?

Reply:
I'm not sure. Let's go with your intuition here (changelog in the past, PRs in the present).